File: Examples/GeodeExampleCSharp (WinForms)/MainWindow.cs
using System;
using System.Windows.Forms;

namespace GeodeExampleCSharp
{
    public partial class MainWindow : Form
    {
        public Extension ExtensionChild;

        public MainWindow()
        {
            InitializeComponent();
        }

        private void MainWindow_Load(object sender, EventArgs e)
        {
            ExtensionChild = new Extension(this);
        }

        private void SendAlertButton_Click(object sender, EventArgs e)
        {
            ExtensionChild.ShowAlert("Hello world!");
        }
    }
}

File: Examples/GeodeExampleVB (WinForms)/MainWindow.vb
Public Class MainWindow
    Public ExtensionChild As Extension

    Private Sub MainWindow_Load(sender As Object, e As EventArgs) Handles Me.Load
        ExtensionChild = New Extension(Me)
    End Sub

    Private Sub SendAlertButton_Click(sender As Object, e As EventArgs) Handles SendAlertButton.Click
        ExtensionChild.ShowAlert("Hello world!")
    End Sub
End Class
